lib/checkPullRequestLabels.js

```diff
@@ -0,0 +1,40 @@
+module.exports.checkPullRequestLabels = async function(context) {
+  var pullRequest = context.payload.pull_request;
+  var pullRequestNumber = pullRequest.number;
+
+  // eslint-disable-next-line no-console
+  console.log('RUNNING LABEL CHECKS ON PULL REQUEST ' + pullRequestNumber + ' ...');
+
+  var labels = pullRequest.labels;
+  var userName = pullRequest.user.login;
+  var projectOwnerLabel;
+  var hasProjectOwnerLabel = labels.some(function(label) {
+    projectOwnerLabel = label.name;
+    return projectOwnerLabel.startsWith('PROJECT');
+  });
+
+  // If the PR has a project owner label, assign the project owner
+  // for the first-pass review of the PR and leave a top-level comment.
+  if(hasProjectOwnerLabel === true) {
+    var labelSubstrings = projectOwnerLabel.split('@');
+    var projectOwner = labelSubstrings[labelSubstrings.length-1].trim();
+
+    var assigneeParams = context.issue({assignees: [projectOwner]});
+    await context.github.issues.addAssigneesToIssue(assigneeParams);
+
+    var commentParams = context.issue({body: 'Assigning @' + projectOwner +
+      ' for the first-pass review of this pull request. Thanks!'});
+    await context.github.issues.createComment(commentParams);
+    return;
+  }
+
+  // If no project owner label is found, close the PR.
+  // Ask the author to add the label and re-open it.
+  var stateParams = context.issue({state: 'closed'});
+  await context.github.issues.edit(stateParams);
+
+  var stateCommentParams = context.issue({body: 'Hi, @' + userName +
+    '. This pull request has been closed since it does have a project ' +
```

Comment:
I looked at the list of labels and as a new contributor, I have no idea what a "project owner label" is. You might want to make this instruction more explicit.

That said ... are you sure it should be "project owner" rather than "project"? I'm not sure how a new contributor is going to know who the project owner is.

Reply:
Okay, I'll look into this. We can redirect them to the gitter channel if they are unable to decide the project label for a PR.

Reply:
Does it look better now? Thanks!

Reply:
"since it **does** have a project label..." ??

Also, it seems to me like you should make this requirement clear in the PR checkbox. Then, in this error message, you can say something like "as mentioned in the PR checkbox list" to draw attention to that list the next time.

Reply:
Done, thanks! Also, I've added a TODO regarding listing this instruction in the PR checkbox. Thanks!